public/ng-views/maps.html

```diff
@@ -19,23 +22,23 @@
         <small>{{ user.name }}</small>
       </i>
     </span>
-  </span>
+  </span> -->
 </div>
 
 
-<div class="topnav right">
+<!-- <div class="topnav right">
   <div class="fa-stack" ng-click="saveCombo=true">
     <i class="fa fa-circle fa-stack-2x"></i>
     <i class="fa fa-arrow-left fa-stack-1x fa-inverse"></i>
-  </div>
-  <div class="fa-stack fa-lg" ng-click="dotSelect = !dotSelect; updateTargetFeature(); overview=false; tabnav=null">
+  </div> -->
+  <!-- <div class="fa-stack fa-lg" ng-click="dotSelect = !dotSelect; updateTargetFeature(); overview=false; tabnav=null">
     <i class="fa fa-circle fa-stack-2x"></i>
     <i class="fa fa-bus fa-stack-1x fa-inverse"></i>
-  </div>
+  </div> -->
 
   <span ng-show="dotSelect">
-    <span class="fa-stack"
-          ng-click="targetCorridor('BH');
+    <span class="fa-stack" 
+          ng-click="targetCorridor('BH'); 
                     tabnav='BH'; overview=true"
           ng-class="tabnav !== 'BH' ? 'dot-unselected' : ''"
           style="color:#0071CA">
```

Comment:
I use styles here because the colors are a dynamic component but in general never do this. Also color changes don't affect UX so they are ok